tests/ProductTest.php

```diff
@@ -284,6 +287,133 @@ public function testProcessInitialData($itemId, $productNameId, $data, $texts)
         }
     }
 
+    public function processVariationMergeKeywordsAndTags()
+    {
+        return [
+            'Texts keywords is set;Tag is of type variation and is set' => [
+                [
+                    'initialData' => [
```

Comment:
Instead of providing an array with keys `initialData` and `variationData`, you could pass them as separate parameters to the test.